Source/ACE/Entity/MovementData.cs

```diff
@@ -250,6 +169,29 @@ public MovementData ConvertToClientAccepted(uint holdKey, CreatureSkill run)
             return md;
         }
 
+        public void SetMovementStateFlag()
+        {
```

Comment:
I have tried to do it the way you requested - but it does not seem to work.   It seems that it should I must have something obviously wrong.